src/TodoListItem.js

```diff
@@ -1,8 +1,9 @@
 import React from "react"
 
 function TodoListItem(props) {
```

Comment:
you can destructure the props in the parameter of a function - `function TodoListItem({ title })`